Frends.SFTP.DownloadFiles/Frends.SFTP.DownloadFiles/Definitions/FileOperations.cs

```diff
@@ -45,9 +45,27 @@ internal static async Task CopyAsync(string source, string remoteFile, bool over
     {
         FileMode fileMode = overwrite ? FileMode.Create : FileMode.CreateNew;
 
-        using Stream sourceStream = new FileStream(source, FileMode.Open, FileAccess.Read);
-        using Stream destinationStream = new FileStream(remoteFile, fileMode, FileAccess.Write);
-        await sourceStream.CopyToAsync(destinationStream, bufferSize: 81920, cancellationToken);
+        var tries = 1;
+        while (tries > 0)

```

Comment:
Can we change this into a less complex logic like `for (var tries = 0; tries <= 1; tries++)`?

Reply:
Removed this altogether because the bug was caused by something else.